<?xml version="1.0" encoding="UTF-8"?>

<launch>

	
	<arg name="target_speed" default="30.0"/>
	<arg name="max_headway" default="60.0"/>

		
	<param name="/max_headway" value="$(arg max_headway)"/>
	<param name="/target_speed" value="$(arg target_speed)"/>
	


	<node pkg="acc_settings" type="acc_settings_node" name="acc_settings_node" output="screen">
    </node>	

	
	

</launch>
